bootstrap/src/io/appium/android/bootstrap/handler/ConfiguratorHandler.java

```diff
@@ -0,0 +1,64 @@
+package io.appium.android.bootstrap.handler;
+
+import com.android.uiautomator.core.Configurator;
+import io.appium.android.bootstrap.AndroidCommand;
+import io.appium.android.bootstrap.AndroidCommandResult;
+import io.appium.android.bootstrap.CommandHandler;
+import org.json.JSONException;
+
+import java.util.Hashtable;
+
+import static io.appium.android.bootstrap.utils.API.API_18;
+
+/**
+ * This handler set {@link com.android.uiautomator.core.Configurator} related settings including
+ * ActionAcknowledgmentTimeout, KeyInjectionDelay, ScrollAcknowledgmentTimeout, WaitForIdleTimeout and WaitForSelectorTimeout
+ * <br/>command example {"cmd" :"action", "params":{"config":"setActionAcknowledgmentTimeout","value":5000}}
+ * <br/>example is the same as invoking {@link com.android.uiautomator.core.Configurator#setActionAcknowledgmentTimeout} to 5 seconds
+ */
+public class ConfiguratorHandler extends CommandHandler {
+    @Override
+    public AndroidCommandResult execute(AndroidCommand command) throws JSONException {
+        if (API_18) {
+            final Hashtable<String, Object> params = command.params();
+            int value = 0;
+            if (params.containsKey("value")) {
+                value = (Integer) params.get("value");
+            }
+            String methodName = ((String) params.get("method"));
+            //TODO: use reflection to invoke method would be more expandable; but Configurator is singleton
+            Configurator configurator = Configurator.getInstance();
+            switch (methodName) {
+                case "setActionAcknowledgmentTimeout":
+                    if (value == 0) // set to default when no value or zero
+                        value = 3000;
+                    configurator.setActionAcknowledgmentTimeout(value);
+                    break;
+                case "setKeyInjectionDelay":
+                    //KeyInjectionDelay default is 0
+                    configurator.setKeyInjectionDelay(value);
+                    break;
+                case "setScrollAcknowledgmentTimeout":
+                    if (value == 0) // set to default when no value
```

Comment:
done in https://github.com/appium/appium-android-bootstrap/pull/25/commits/4a35db8c76afa5fcaffb92a5b9adbc0b4c042f49